src/internal/direction.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * Determine if the direction of a given element is left-to-right
+ * or right-to-left. The is the JavaScript equivalent of the
+ * with-direction mixin that implements the dir() pseudoclass.
+ */
+export function isRtl(element: HTMLElement | EventTarget) {
+  let isRtl = false;
```

Comment:
Nitpick: this could be simplified a little bit without the need of the `let` declaration:
```
if (element instanceof HTMLElement) {
  return getComputedStyle(element).direction === 'rtl';
}

return false;
```